docs/content/odf/buckets-external-access.md

```diff
@@ -138,28 +143,47 @@ Assuming you have configured the environment variables described in
 the "Extract bucket information" section, create a MinIO alias
 definition for your bucket:
 
+
 ```
-mcli alias set example-obc $BUCKET_ENDPOINT $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY
+mc alias set example-obc $BUCKET_ENDPOINT $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY
```

Comment:
Let's add a note, that `mc` refers to MinIO client not Midnight Commander and link to this: https://github.com/minio/mc/blob/master/CONFLICT.md

I don't care about the rename.. I just don't want people to be confused. Some linux distros package MinIO client as `mcli`, some as `mc` :shrug: